apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -221,13 +221,14 @@ function enqueue_coblocks_gallery_scripts() {
  * Load Blog Posts block.
  */
 function load_blog_posts_block() {
-	$slug          = 'newspack-blocks/newspack-blocks.php';
+	// Use regex instead of static slug in order to match plugin installation also from github, where slug may contain (HASH|branch-name).
+	$slug_regex    = '/newspack-blocks(-[A-Za-z0-9-]+)?\/newspack-blocks\.php/';
 	$disable_block = (
 		( defined( 'WP_CLI' ) && WP_CLI ) ||
 		/* phpcs:ignore WordPress.Security.NonceVerification */
 		( isset( $_GET['action'], $_GET['plugin'] ) && 'activate' === $_GET['action'] && $slug === $_GET['plugin'] ) ||
```

Comment:
With the above change `$slug` is undefined. Let's replace that with a `preg_match` call as well

Reply:
wow, shouldn't have missed that. fixed